_data/faq.yml

```diff
@@ -0,0 +1,22 @@
+    - {question: "Why should I participate in a hackathon?", answer: "Hackathons are a great way to learn new skills that will be useful in your every day work as a researcher. It is also a very good way to meet and collaborate with new people."}
+-
+  group: "OHBM Hackathon"
+  questions:
+    - {question: "What is the OHBM Hackathon?", answer: "OHBM hackathons are events for students, engineers and researchers who work with MRI, PET, EEG, and MEG to come together to learn and collaborate. The OHBM hackathons have 2 components: a <b>traintrack</b> which is a series of workshops, tutorials and demos where you can learn new skills and a <b>hacktrack</b> where you can propose a project you need help with, or help others with their project."}
+    - {question: "How will the first ever virtual OHBM Hackathon be different?", answer: "The open-science special interest group of OHBM wants to rise to the challenge we are all facing and create a decentralized online event to foster global engagement and collaboration across time zones and geographical boundaries. For 3 days, no matter where you are on the planet, there will be <b>traintrack</b> sessions happening where you can learn new things and <b>hacktrack</b> projects that you can join."}
+    - {question: "Do I need to register to OHBM to attend the OHBM hackathon?", answer: "No. The main conference and the hackahton are different events and you can choose to only attend only the hackathon."}
+    - {question: "Why will there be several hubs?", answer: "While anyone around the world can join an online event, we wanted to make it easier for people to join during more or less normal working hours for them."}
+    # - {question: "Is the OHBM Hackathon run by an institute/company/individuals?", answer: "OHBM"}
+    # - {question: "May I be involved with the OHBM hackathon team?", answer: "OHBM"}
+    # - {question: "Is there any funding available to hackathon attendees?", answer: "OHBM"}
+-
+  group: "Attendees"
+  questions:
+    # - {question: "May I pitch a project? Do I have to? Which procedure do I have to follow?", answer: " "}
+    # - {question: "Am I eligible to attend a Distributed Hackathon event?", answer: " "}
+    - {question: "How do I join a project at a hackathon? ", answer: "In the weeks leading up to the hackathon, project leaders will submit a description of their project. The projects will be listed on a github repository and on the website. So you can start having a look at what projects are lined up and think about which one you want to join. Then at the hackathon each project leader has to pitch his project at the beginning of the event. Then there will be some time for you to chat with the project leaders before you 'decide' which projects to join. But it is a very flexible thing: if you are interested by different projects and wants to do a bit of both: it is OK."}
```

Comment:
should we perhaps encourage newcomers to only join one project?

Reply:
yes good point.
will add this as a comment in the google doc.

I think that the FAQ page is "good enough" in terms of look.

Now I will focus on the content.